stdlib/public/stubs/LibcShims.cpp

```diff
@@ -183,6 +183,9 @@ char * _Nullable *swift::_stdlib_getEnviron() {
 }
 #endif
 
+// not Windows
+#endif
```

Comment:
I realize that this is not your fault, but, while you are here, can you change this please?  This should use the following form rather than the preceding `// not windows`.

    #endif // defined(_WIN32) && !defined(__CYGWIN__)

Reply:
Sure. I've added a comment there. I also thought it made more sense inverted 

```
// !(defined(_WIN32) && !defined(__CYGWIN__))
```

since the `#endif` is actually the end of the `else` block, but if you object to that I'll change it to what you originally had.